src/Pages/Overview/DisplayProfile.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
```

Comment:
I understand that the intention here is to try and simplify things, but this splits functions between two files making things confusing. Instead,

1. make `displayBoard` its own react component. (send parameters via props)
1. move all non html rendering functions back in their respective classes
1. take the `marks` function and make it its own react component. (send parameters via props)

Reply:
I will move all the SVG into a js file and return functions instead. I think this is more clean